spec/CloudCode.spec.js

```diff
@@ -1163,6 +1163,53 @@ describe('Cloud Code', () => {
     );
   });
 
+  it('test save triggers get user roles', async () => {
+    let beforeSaveFlag = false,
+      afterSaveFlag = false;
+    Parse.Cloud.beforeSave('SaveTriggerUserRoles', async function (req) {
```

Comment:
Please change syntax to this for all other occurrences as well:
```suggestion
    Parse.Cloud.beforeSave('SaveTriggerUserRoles', async req => {
```